examples/light-switch-app/nrfconnect/README.md

```diff
@@ -0,0 +1,593 @@
+ - < switch id > &mdash; node_id of light_switch-app used in commissioning
+
+ 
+1) Add ACL (Access Control List) in Lighting-App device:
+
+To give the lighting endpoint permissions to receive commands from light switch an ACL should be written. An ACL should contain information about all clusters which can be called by light-switch.
+
+Command: 
+
+    $ chip-tool accesscontrol write acl '[{"fabricIndex": 1, "privilege": 5, "authMode": 2, "subjects": [112233], "targets": null}, {"fabricIndex": 1, "privilege": 3, "authMode": 2, "subjects": [<switch id>], "targets": [{"cluster": 6, "endpoint": 1, "deviceType": null}, {"cluster": 8, "endpoint": 1, "deviceType": null}]}]' <lighting id> 0
+
+
+Where:  
+__{"fabricIndex": 1, "privilege": 5, "authMode": 2, "subjects": [112233], "targets": null}__ is an ACL for communication with chip-tool 
+
+__{"fabricIndex": 1, "privilege": 3, "authMode": 2, "subjects": [< switch id >], "targets": [{"cluster": 6, "endpoint": 1, "deviceType": null}, {"cluster": 8, "endpoint": 1, "deviceType": null}]}__ is an ACL for binding (cluster nr 6 is a _onoff_ cluster and nr 8 is a _LevelControl_ cluster).
+ 
+To check if ACL was added use command:
+
+    $ chip-tool accesscontrol read acl <lightning id>  0 
+
+ 2) Add binding table:
+    
+Command for unicast binding:
+
+    $ chip-tool binding write binding '[{"fabricIndex": 1, "node": <lighting id>, "endpoint": 1, "cluster": 6}, {"fabricIndex": 1, "node": <lighting id>, "endpoint": 1, "cluster": 8}]' <light switch id> 1
+
+Where:  
+__{"fabricIndex": 1, "node": <lighting id>, "endpoint": 1, "cluster": 6}__ is a binding for __onoff__ cluster 
+
+__{"fabricIndex": 1, "node": <lighting id>, "endpoint": 1, "cluster": 8}__ is a binding for __LevelControl__ cluster. 
+
+To use light switch without brightness dimmer apply only first binding command.
+
+ If Light switch device is rebooting a binding table is restoring from flash memory and device tries to bind known lighting-app device.
+
+
+After binding process a lighting device can be controlled by light-switch in two ways - by pressing [button 2](#device-ui) or using matter command-line interface. 
+
+To enable matter CLI a light_switch-app example should be compiled with additional option __-DCONFIG_CHIP_LIB_SHELL=y__. Run the following command with _build-target_ replaced with the build target name of the Nordic Semiconductor kit you are using (for example _nrf52840dk_nrf52840_):
+
+    $ west build -b build-target -- -DCONFIG_CHIP_LIB_SHELL=y
+
+Use these commands to control lighting app via Matter CLI:
+
+    uart:~$ switch onoff on     : sends unicast On command to bound device
+    uart:~$ switch onoff off    : sends unicast Off command to bound device
+    uart:~$ switch onoff toggle : sends unicast Toggle command to bound device
+
+Check the [CLI tutorial](../../../docs/guides/nrfconnect_examples_cli.md) to
+learn how to use other command-line interface of the application.
```

Comment:
This is not part of the binding process, so these lines need a new section.
One way of doing this is adding a header in line 573, and then after an introductory line add two subsections -- one about interacting with Button 2 and describing the LED changes, and one about using the CLI. You need to provide specific steps that showcase the testing actions, reaction of LEDs, and the final result. It cannot be something like lines 580-584, where the user is just provided with a list of commands and no result is described.